frontend/src/model/Tile.ts

```diff
@@ -51,6 +74,24 @@ class Tile {
       right: prevEdges.top,
       top: prevEdges.left,
     };
+    switch (this.rotation) {
+      case 0: {
+        this.rotation = 90;
+        break;
+      }
+      case 90: {
+        this.rotation = 180;
+        break;
+      }
+      case 180: {
+        this.rotation = 270;
+        break;
+      }
+      default: {
+        this.rotation = 0;
+        break;
+      }
```

Comment:
wouldn't it be shorter if we add 90 to this.rotation value and after addition check if 360 then this.rotation = 0?